### DISCUSSION TITLE
Toggle connection state based on time of day

### DISCUSSION THREAD

**Discussion by Torinth** [bitfocus/companion#3024] [Q&A] [answered]
Hello, I am using the TP Link Tapo module to control some outlets.  It has a polling configuration option but i would rather not have that on.  Without it over the weekend companion loses its ability to communicate with the outlets due to a timeout.  I found that if i just open the connections and disable/reenable them they get their connection back and work again.

So what i am hoping there is a solution for is to have some way to use triggers to say "each day at 4am disable then re enable these connections" so that way they always have a fresh connection and wont lose ability to communicate.  Or would it work do yall think to have like windows task scheduler item to just straight quit companion and then re open it?  Not ideal but might would work?

Thanks very much.

**Comment by Julusian** [comment, marked as answer]
This is achievable with a trigger.
Triggers can act at a certain time each day, and can use any action in companion such as the one to enable/disable connections

**Comment by Torinth** [comment]
Ah my bad. I had looked at triggers before but I didn’t think to look at the internal selections. Thank you
________________________________
